conduit_test/CHANGELOG.md

```diff
@@ -12,7 +12,7 @@
 
 - Alpha release for Conduit NNBD included
 
-# Aqueduct
+# Conduit
```

Comment:
@bsutton I am considering just removing the changelogs from Aqueduct downward. Can't think of a strong reason to keep them.